src/webextension/list/manage/components/account-status.js

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import PropTypes from "prop-types";
+import React from "react";
+import { connect } from "react-redux";
+
+import Button from "../../../widgets/button";
+
+export function AccountStatus({email, onClick, ...props}) {
```

Comment:
I imagine this is just WIP, but you're not using `onClick` or `props`. You could probably remove the latter entirely (and add the former to the `Button`).

Reply:
This is weird. Not sure why ESLint wouldn't have detected the unused arguments here.